### PR TITLE
EAGLE-106 Use nvd3 replace flot for UI chart

### PR DESCRIPTION
**Pull request by zombieJ**
flot is no maintenance for more than 1 year. Some feature need additional work. Use nvd3 to replace.

### CODE REVIEW

File: eagle-webservice/src/main/webapp/app/partials/dam/nvd3.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<style>
```
Comment (haoch) [inline eagle-webservice/src/main/webapp/app/partials/dam/nvd3.html]: License header is missing.

**Comment by zombieJ** [comment]
This is dev page. Add cleanup commit to remove it.

**Comment by haoch** [comment]
LGTM

**Comment by qingwen220** [comment]
:+1:

**Pull request closed without merging**